### DISCUSSION TITLE
[🚨] Bot compliance warning

### DISCUSSION THREAD

**Discussion by showierdata9978** [MeowerBots/.github#7] [Announcements]
All repos must follow the readme rules within 48 hours of this post, or face archival by me or @MikeDev101. 

@MeowerBots/bot-teams 
@MeowerBots/botlibs

**Comment by theycallhermax** [comment]
ok

**Comment by showierdata9978** [comment]
@MeowerBots/bot-teams 
@MeowerBots/botlibs 

Rules have been expanded on

**Reply by theycallhermax** [reply]
ok

**Comment by JoshAtticus** [comment]
It's compliance
